applications/satellite-imagery-geotiff.ipynb

```diff
@@ -0,0 +1,479 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import xarray as xr\n",
+    "red = xr.open_rasterio('red.tif', chunks={'band': 1, 'x': 1024, 'y': 1024})\n",
+    "nir = xr.open_rasterio('nir.tif', chunks={'band': 1, 'x': 1024, 'y': 1024})\n",
+    "nir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each dataset's data is a Dask array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "dask.array<open_rasterio-80c6f5cb2f8ebf72adc46b64f31017d1<this-array>, shape=(1, 7841, 7721), dtype=uint16, chunksize=(1, 1024, 1024)>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "red.variable.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optional: create a Dask client\n",
+    "\n",
+    "You can start a Dask client to monitor execution with the dashboard."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table style=\"border: 2px solid white;\">\n",
+       "<tr>\n",
+       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
+       "<h3>Client</h3>\n",
+       "<ul>\n",
+       "  <li><b>Scheduler: </b>inproc://10.30.139.219/23709/1\n",
+       "  <li><b>Dashboard: </b><a href='http://localhost:8787/status' target='_blank'>http://localhost:8787/status</a>\n",
+       "</ul>\n",
+       "</td>\n",
+       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
+       "<h3>Cluster</h3>\n",
+       "<ul>\n",
+       "  <li><b>Workers: </b>1</li>\n",
+       "  <li><b>Cores: </b>4</li>\n",
+       "  <li><b>Memory: </b>4.09 GB</li>\n",
+       "</ul>\n",
+       "</td>\n",
+       "</tr>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<Client: scheduler='inproc://10.30.139.219/23709/1' processes=1 cores=4>"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import dask\n",
+    "from dask.distributed import Client\n",
+    "client = Client(processes=False)\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An example calculation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<xarray.DataArray (band: 1)>\n",
+       "array([481941846348], dtype=uint64)\n",
+       "Coordinates:\n",
+       "  * band     (band) int64 1"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "red.sum(dim=['x', 'y']).compute()"
```

Comment:
Do this operation has sense in some cases? Do we want to keep it or give a more meaningfull example?

Reply:
Fair point, maybe for this case it would be more realistic to compute a windowed mean, resampling the image. I kept this mostly because it has a simple execution plot in the dashboard.

Reply:
Deleted this example